proposal/README.md

```diff
@@ -85,45 +85,109 @@ This is a lower priority option as most entities in flyte are immutable and do n
 # Details of each resource
 
 ## Projects
-Support
+Projects are top level entity in Flyte. You can fetch multiple projects or one project using the CLI. Think about projects like namespaces.
+
  - create
+```bash
+$ flytectl create projects --name "Human readable Name of project" --id project-id --labels key=value --labels key=value --description "long string"
+Alternatively
+$ flytectl create project -f project.yaml
+```
+
+```yaml
+project.yaml
+name: Human readable project name
+id: project-x
+labels:
+  - k: v
+  - k1: v1
+description: |
+  Long description
+```
  - get
+```bash
+$ flytectl get projects [project-name] [-o yaml | -o json | default -o table]
+```
  - update
+```bash
+$ flytectl update projects --id project-x ...
+# You can only update one project at a time
+```
 
 ## Tasks
-Support
- - create
  - get
+```bash
+$ flytectl get tasks [task-name] [-o yaml | -o json | default -o table] [--filters...] [--sort-by...] [--selectors...]
+```
+ - create
  - update
 
 ## Workflows
 Support
- - create
  - get
+```bash
+$ flytectl get workflows [workflow-name] [-o yaml | -o json | default -o table] [--filters...] [--sort-by...] [--selectors...]
+```
+ - create
  - update
 
 ## Launch Plans
 Support
- - create
  - get
+```bash
+$ flytectl get launch-plans [launchplan-name] [-o yaml | -o json | default -o table] [--filters...] [--sort-by...] [--selectors...]
+```
+ - create
  - update
 
 ## Execution
-Support
- - create
+Create or retrieve an execution. 
  - get
+Get all executions or get a single execution.
+```bash
+$ flytectl get execution [exec-name] [-o yaml | -o json | default -o table] [--filters...] [--sort-by...] [--selectors...]
```

Comment:
would be nice to see a `--raw` output option that would emit content pipe-able into another command (like delete).

Reply:
I think we should flesh it out more and basically you just want the name. This can be added easily as output formatter module is fairly standalone